### PR TITLE
Prefix sum thresholds

### PR DESCRIPTION
**Pull request by tewaro**

### CODE REVIEW

File: libgalois/include/galois/graphs/LS_LC_CSR_Graph.h [hunk 1]
```diff
@@ -121,7 +121,13 @@ class LS_LC_CSR_Graph : private boost::noncopyable {
 
   // Compute the prefix sum using the two level method
   void computePrefixSum() {
-    m_pfx.computePrefixSum(m_vertices.size());
+    constexpr uint64_t PARALLEL_PREFIX_SUM_VERTEX_THRESHOLD =
+        static_cast<uint64_t>(1) << 30;
+    if (m_vertices.size() > PARALLEL_PREFIX_SUM_VERTEX_THRESHOLD) {
+      m_pfx.computePrefixSumSerially(m_vertices.size());
```
Comment (meyerzinn) [inline libgalois/include/galois/graphs/LS_LC_CSR_Graph.h:127]: why is this not just internal to `computePrefixSum`

Comment (meyerzinn) [reply libgalois/include/galois/graphs/LS_LC_CSR_Graph.h:127]: Since presumably it would apply to any user of the `PrefixSum` utility.

**Review by meyerzinn — CHANGES_REQUESTED** [review]

File: libgalois/include/galois/graphs/LS_LC_CSR_Graph.h (hunk 1 above)
Comment (tewaro) [reply libgalois/include/galois/graphs/LS_LC_CSR_Graph.h:127]: some input sizes for things like vectors of floats would pick a much smaller threshhold. More complex transmute functions result in  much smaller values of the prefix_sum threshold.

Comment (meyerzinn) [reply libgalois/include/galois/graphs/LS_LC_CSR_Graph.h:127]: I see. I guess that makes sense.

**Review by meyerzinn — APPROVED** [review]

**Pull request merged**